### DISCUSSION TITLE
20240703盤勢

### DISCUSSION THREAD

**Discussion by ChiuAlfredo** [ChiuAlfredo/options_day_data#3] [General]
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/59f98e98-06db-4223-96c6-a86dd34a4b13)
成交量最多的不是在價平

**Comment by Will150143** [comment]
![image](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/7d7d9d15-9fb7-4ee4-9644-5033cacc2130)
價平以上一檔買賣交易量持平 是分水嶺

**Reply by ChiuAlfredo** [reply]
什麼意思啊😀看不太懂

**Reply by Will150143** [reply]
拿這張圖來說 23000差不多是價平 23050也就是價平+1檔時是買權成交量差不多等於賣權成交量的時候

**Comment by Will150143** [comment]
![image](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/d12cccb9-3c4a-468f-9005-31022a911475)
* 賣價微大於買價
* 價內的買賣價差較價外大

**Reply by ChiuAlfredo** [reply]
這是正常狀況吧，所有都會報價都是這樣，賣價=買價會直接成交，這裡是委賣跟委賣

**Reply by Will150143** [reply]
反正我就看到啥就放上來 因為我不知道

**Comment by Will150143** [comment]
![image](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/9b5667e2-d70d-4b25-ae4c-828b9875fc8a)
價外的成交量比價內多不少

**Comment by ChiuAlfredo** [comment]
# 策略

期貨兩次穿越k值選擇權價格相差大

![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/8746b6ec-8fd8-4279-821b-81733c7bee7f)
期貨兩次穿越23100

![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/ff06e1a5-dab9-4cb2-9156-07d58afb50c0)
23100的選擇權價格

![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/37ce1a45-df74-4d62-acb7-6640dfcd72ee)
加權指數都沒有穿越

**Reply by ChiuAlfredo** [reply]
檢驗同樣期貨價格時，買方力道是否降低 ，k=期貨 出現多次突破時，每次選擇權的頭部價格是否一致。
如過三個頭部是向下，買方力道減弱，買空
頭部向上，買方力道強，買入

**Comment by ChiuAlfredo** [comment]
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/37623d1b-9f99-4a66-967b-5e1f42de2844)
1點過接近理論價

**Reply by ChiuAlfredo** [reply]
1:00
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/c219d184-2999-42a8-aa29-d7ccbcefae02)
1:02
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/d2fe8f6a-b234-4e1c-a7bd-379aa07f7ef0)

差兩分鐘，價差1倍

**Comment by Will150143** [comment]
![12 40](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/53c4f736-5f6e-49e6-8d27-ed9ae2c1f9d1)
12.40
![12 42](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/fca51e82-00d3-493c-bf48-b0410e4a2f17)
12.42
23050P（價外)漲

**Reply by ChiuAlfredo** [reply]
12:40指數
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/c6c455f9-1d67-4dab-8e66-84585b9fb2eb)

12:42指數
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/bed41985-77f0-4986-9f68-31b9794f442b)

**Reply by ChiuAlfredo** [reply]
12:41 期貨
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/5e86b038-f272-457d-a3b5-f09d892f9a98)

**Reply by ChiuAlfredo** [reply]
只有幾單上漲，但是馬上下跌
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/777295db-4e7d-42a8-978e-406f32ca927b)

**Reply by ChiuAlfredo** [reply]
> 只有幾單上漲，但是馬上下跌 ![image](https://private-user-images.githubusercontent.com/91011574/345341045-777295db-4e7d-42a8-978e-406f32ca927b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwNzIwODgsIm5iZiI6MTcyMDA3MTc4OCwicGF0aCI6Ii85MTAxMTU3NC8zNDUzNDEwNDUtNzc3Mjk1ZGItNGU3ZC00MmE4LTk3OGUtNDA2ZjMyY2E5MjdiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA0VDA1NDMwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4NjA4ZDNkZmIxY2I1NjFjMWQwYWYwOTA1MTZkNzVkMTllYzkzMzVmYzQzNDFlOTA2ZTFkM2JjZjM3NDVlY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4zUlPcC6rbJ6Tu6AB1_6yufzgYrB9y2VsCpyiiwfP0U)

在價格0.4時，掛0.1
成交後馬上賣出（可能賣價為0.2 or 0.3)

**Comment by ChiuAlfredo** [comment]
大盤在最後一個時刻上漲
[20240730MI_5MINS_INDEX.csv](https://github.com/user-attachments/files/16085391/20240730MI_5MINS_INDEX.csv)

![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/f3771910-ff8e-4c6e-9a60-2db8f4565be3)

台積電最後漲了3點
![image](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/c0e31fd9-affc-4526-826c-905c5d9555db)


但是根據選擇權結算價，這部分不會影響到結算價格

結算公式：
股價指數類期貨及選擇權契約之最後結算價，以最後結算日（到期日）臺灣證券交易所及中華民國證券櫃檯買賣中心當日交易時間收盤前30分鐘內標的指數之算術平均價訂之。
--
  | 前項標的指數之算術平均價，係以前項交易時間內臺灣證券交易所、中華民國證券櫃檯買賣中心、臺灣指數股份有限公司、富時國際有限公司(FTSE)或臺灣指數股份有限公司委請臺灣證券交易所每次揭示之標的指數(交易時段13:00(不含)至13:25(含)，加計最後一筆收盤指數)，採簡單算術平均計算，並取至最接近各期貨契約報價最小升降單位整數倍之數值訂之

**Reply by ChiuAlfredo** [reply]
![IMG_1719](https://github.com/ChiuAlfredo/options_day_data/assets/91011574/72b50da5-0e75-45f7-8452-4c69d8612d28)
